### ISSUE TITLE
nn.functional No-batch-dim support should have OpInfo examples

### ISSUE THREAD

**Issue by zou3519** [pytorch/pytorch#72672]
### 🚀 The feature, motivation and pitch

Filing this so we can track the issue. @jbschlosser tells me that no-batch-dim (https://github.com/pytorch/pytorch/issues/60585) testing is done with ModuleInfo and not OpInfo. However, this means that subsystems that only use OpInfos to test their semantics may be unaware of the new changes. We should update OpInfos with no-batch-dim examples like how we added ModuleInfo testing for them.
